packages/android_intent_plus/example/test_driver/android_intent_plus_e2e.dart

```diff
@@ -70,7 +70,7 @@ void main() {
     final intent = AndroidIntent(
       action: 'action_view',
       package: 'io.flutter.plugins.androidintentexample',
-      componentName: 'io.flutter.embedding.android.FlutterActivity',
+      componentName: 'io.flutter.plugins.androidintentexample.MainActivity',
```

Comment:
this was a bug unrelated with the null safety part, but necessary to pass the tests.